reference/cookie_consent.yml

```diff
@@ -0,0 +1,94 @@
+openapi: 3.0.1
+servers:
+  - url: 'https://{$$.env.store_domain}/api/storefront'
+info:
+  title: Storefront Cookie Consent API
+  description: Storefront API for specifying consent to particular categories of cookies for compliance purposes.
+  version: Storefront
+tags:
+  - name: Customers
+paths:
+  /consent:
+    post:
+      summary: Specify Shopper Cookie Consent Preferences
+      tags:
+        - Customers
+      responses:
+        '200':
+          description: Consent Settings Saved
+        '400':
+          description: Invalid input
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ConsentPreferences'
+            examples:
+              Example:
+                value:
+                  allow:
+                    - 2
+                    - 3
+                  deny:
+                    - 4
+        description: Data sent to the customer endpoint when creating a customer during checkout
+        required: true
+      description: |-
```

Comment:
I think we need to fix the formatting a bit. Consent categories are not displaying as a list. Also, please remove passive voice where possible.